scripts/run-integration-tests.sh

```diff
@@ -138,10 +138,25 @@ cp "$BASE_CONFIG_PATH" "$TEST_CONFIG_PATH"
 sed -i'.bak' "s,602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni,$IMAGE_NAME," "$TEST_CONFIG_PATH"
 sed -i'.bak' "s,$STABLE_IMAGE_VERSION,$TEST_IMAGE_VERSION," "$TEST_CONFIG_PATH"
 
-echo "*******************************************************************************"
-echo "Deploying CNI with image $IMAGE_NAME"
 export KUBECONFIG=$KUBECONFIG_PATH
-kubectl apply -f "$TEST_CONFIG_PATH"
+ADDONS_CNI_IMAGE=$($KUBECTL_PATH describe daemonset aws-node -n kube-system | grep Image | cut -d ":" -f 2-3 | tr -d '[:space:]')
+
+echo "*******************************************************************************"
+echo "Running integration tests on default CNI version, $ADDONS_CNI_IMAGE"
+echo ""
+pushd ./test/integration
+GO111MODULE=on go test -v -timeout 0 ./... --kubeconfig=$KUBECONFIG --ginkgo.focus="\[cni-integration\]" --ginkgo.skip="\[Disruptive\]" \
+    --assets=./assets
+TEST_PASS=$?
+popd
+
+echo "*******************************************************************************"
+echo "Updating CNI to image $IMAGE_NAME:$TEST_IMAGE_VERSION"
+$KUBECTL_PATH apply -f "$TEST_CONFIG_PATH"
+
+echo "Sleping for 110s"
+echo "TODO: Poll and wait for updates to complete instead!"
+sleep 110
```

Comment:
why 110 seconds? is that a magic value? :)

Reply:
Yes, for now. The test cluster has 3 nodes, and the rolling update is using the default grace period of 30s, so it takes around 30s to shut down each node, and another 5-10 seconds to start up. It's ok to start the test once all old ipamd pods have shut down.